samples/tls/monitoring/README.md

```diff
@@ -4,46 +4,10 @@
 
 This sample demonstrates how to deploy monitoring components with TLS enabled.
 
-All components have TLS enabled on ingress. Due to limitations in the
-underlying Helm charts, some components might not have TLS enabled in-cluster.
-See the **Limitations and Known Issues** section below for details.
-
-## Ingress
-
-If you use this sample for HTTPS for ingress, the following secrets must be
-manually populated in the `monitoring` namespace (or `MON_NS` value)
-**BEFORE** you deploy cluster monitoring.
-
-* kubernetes.io/tls secret - `prometheus-ingress-tls-secret`
-* kubernetes.io/tls secret - `alertmanager-ingress-tls-secret`
-* kubernetes.io/tls secret - `grafana-ingress-tls-secret`
-
-Generating these certificates is outside the scope of this example. However,
-you can use the process documented in ["Configure NGINX Ingress TLS for SAS Applications"](https://go.documentation.sas.com/?cdcId=sasadmincdc&cdcVersion=default&docsetId=calencryptmotion&docsetTarget=n1xdqv1sezyrahn17erzcunxwix9.htm&locale=en#n0oo2yu8440vmzn19g6xhx4kfbrq)
-in SAS Viya Administration documentation and specify the `monitoring` namespace.
+If you enable TLS, communications between the user and the monitoring components 
```

Comment:
Setting TLS_ENABLE only enables TLS between in-cluster monitoring components.

The confusing part is that when using NodePorts (the default), the user->monitoring component communication will also be TLS-enabled since NodePorts are a direct line to the internal components without ingress in-between.

Reply:
@BryanEllington are you saying browser to Grafana/Prometheus/AlertManager traffic requires HTTPS by default?  That's not what I've seen in my deployments (all connections are HTTP and shown as "Not secure" in my browser).

Reply:
Intro sentences changed to "If you enable TLS, in-cluster communications between monitoring components use TLS. 

If you enable TLS and are using nodeports, connections between the user and monitoring components also use TLS."

Reply:
That's true but just reflects the need for TLS to communicate with the monitoring objects rather than any "extra"/"additional" (configuration) work.  Just mentioning that as clarification, not necessarily as something that needs to go into the doc.